### PR TITLE
Change team page layout

### PR DESCRIPTION
**Pull request by sauhardar**
Change the layout of the team page to divide members into different sections based on team (eboard, searchneu, visualizer, etc.)

### CODE REVIEW

File: src/components/teamPhotos.js [hunk 1]
```diff
@@ -127,15 +128,44 @@ const IconWrapper = styled.div`
...
+  }
+`
+
 const Profile = ({ member, percent }) => {
   const { name, team, socialMedia } = member
-  const { email, linkedIn } = socialMedia
+  const { email, linkedIn, portfolio } = socialMedia
   const { name: teamName, role } = team
-  // Prevents repetitive code by storing content in one place.
+  console.log(portfolio)
```
Comment (dajinchu) [inline src/components/teamPhotos.js]: don't forget to clean up

File: src/components/teamPhotos.js [hunk 2]
```diff
@@ -150,22 +180,50 @@ const Profile = ({ member, percent }) => {
...
+            <Profile member={member} key={member.name} percent={i / length} />
+          )
+        })}
+      </TeamMembersWrapper>
+    </div>
+  )
+}
+
+const groupMembers = members => {
+  const grouped = _.groupBy(members, "team.name")
```
Comment (dajinchu) [inline src/components/teamPhotos.js]: Nice!

File: src/content/team/Teams/teams.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+const teams = {
```
Comment (dajinchu) [inline src/content/team/Teams/teams.js]: I can't figure out where this object is used. Is it needed?

Comment (sauhardar) [reply src/content/team/Teams/teams.js]: Yeah I was thinking about a way to store the team names as constants somewhere, but you can't import variables into JSON files. I'll delete this file. Do you have any ideas?

Comment (dajinchu) [reply src/content/team/Teams/teams.js]: I think what you have with the groupby is good

**Pull request merged**